### DISCUSSION TITLE
contributing to journey book

### DISCUSSION THREAD

**Discussion by collab-community** [collab-community/journey-book#42] [Q&A] [answered]
@KendallDoesCoding @vikasganiga05 

-- Now should i ask pepole for writing there journey (introduction) in journey book file ?
-- If yes then kindly update contributing guidelines in readme file , and I am going to update moto of this project , like I have thought something exciting for  present it to folks 😄

**Comment by KendallDoesCoding** [comment, marked as answer]
> @KendallDoesCoding @vikasganiga05
> 
> -- Now should i ask pepole for writing there journey (introduction) in journey book file ? -- If yes then kindly update contributing guidelines in readme file , and I am going to update moto of this project , like I have thought something exciting for present it to folks 😄

Yes. please ask them to create their journey by a introduction in the directory, it would be good if you, @Adarsh-jaiss , @vikasganiga05 , @TechStudent11 could start off first, before sharing to other people though.

Shouldn't the contributing guidelines go in the CONTRIBUTING.md file within the .github directory?

For now, any journey's that come in we'll do the work of adding on the website, but later the users will have to add the .json and stuff once we merge the website to this repo, which is going to be pretty soon.

**Reply by KendallDoesCoding** [reply]
I have reached out to a couple Youtubers, friends asking them to share their journey, earlier today as well, hopefully they share their journey :)

**Reply by Adarsh-jaiss** [reply]
okay, i am adding my introduction :)

**Reply by KendallDoesCoding** [reply]
cool

**Reply by KendallDoesCoding** [reply]
> okay, i am adding my introduction :)

hi @Adarsh-jaiss , any update on this?
@vikasganiga05 you too please add your journey

@TechStudent10 - you too are free to add a journey, but as you stated in ur discussion you don't want to do so.

**Reply by TechStudent10** [reply]
YOU KEEP PINGING MY ALT

**Reply by Adarsh-jaiss** [reply]
@KendallDoesCoding actually I am on my vacation🏖 and I don't have my laptop with me, so that's why I haven't done that yet

**Comment by KendallDoesCoding** [comment]
ooh ok, no worries. enjoy your vacation!
